code/modules/mining/equipment/explorer_gear.dm

```diff
@@ -22,8 +22,8 @@
 	desc = "An armoured hood for exploring harsh environments."
 	icon_state = "explorer"
 	body_parts_covered = HEAD
-	flags_inv = HIDEHAIR|HIDEFACE|HIDEEARS
-	flags_prot = HIDEHAIR|HIDEFACE
```

Comment:
don't really like this, will kill off any burnable masks and goggles you use

Reply:
explorer gas masks are fireproof and protect goggles

Reply:
also normal gasmasks are as well and there's no shortage of those, plus 4 extra explorer masks in the suit storage units on lavaland anyway